app/views/alert_mailer/daily_alert.text.haml

```diff
@@ -15,8 +15,8 @@
   \---
   \
 \
-Visit #{notify_link('https://teaching-vacancies.service.gov.uk', t('app.title'))} - the free service for schools in England to list teaching roles and for jobseekers to find them.
+Visit #{notify_link('https://teaching-vacancies.service.gov.uk', t('app.title'))} -  the free service for schools in England to list teaching roles and for jobseekers to find them.
```

Comment:
Looks like there is an additional space on this line, not sure if that was intentional.

Reply:
Thanks for spotting this. 
Not intentional. 
Can see that this was pushed as part of a WIP commit: ea6fda, so that someone else could continue where I left off since I had to quickly work on another feature.